src/content/docs/en/guides/astro-db.mdx

```diff
@@ -654,6 +655,68 @@ When you're ready to deploy, see our [Deploy with a Studio Connection guide](#de
 
 </ReadMore>
 
+## LibSQL
+
+<p><Since pkg="@astrojs/db" v="0.14.0" /></p>
+
+Astro DB can connect to any LibSQL server from any platform that exposes the Hrana protocol of the server, or can be self-hosted.
+
+To connect Astro DB to a LibSQL database, set the following environment variables:
+- `ASTRO_DB_REMOTE_URL`: the connection URL to your LibSQL server
+- `ASTRO_DB_APP_TOKEN`: the auth token to your LibSQL server
+
+The [commands for deploying and pushing changes](#deploy-with-a-studio-connection) to your database are the same when using LibSQL as when connecting to a Studio hosted database. However, both of your environment variables need to be set locally when running commands with the `--remote` option like `astro build` and `astro db push`.
```

Comment:
```suggestion
The [commands for deploying and pushing changes](#deploy-with-a-studio-connection) to your database are the same when using LibSQL as when connecting to an Astro Studio hosted database. However, both of your environment variables need to be set locally when running commands with the `--remote` option like `astro build` and `astro db push`.
```

Possibly make this a bit more explicit for those who may be linking to this specific section.

Reply:
Love it, thanks!